admin/js/wi-volunteer-management-block.js

```diff
@@ -124,10 +127,24 @@
 		},
 	} );
 
-	// Hide the volunteer opportunities block when editing a volunteer opportunity post
-	wp.domReady( function() {
-		if( wp.data.select( 'core/editor' ).getCurrentPostType() === 'volunteer_opp' ) {
-			wp.blocks.unregisterBlockType( 'wired-impact-volunteer-management/volunteer-opps' );
+	// We have to subscribe to changes because on domReady the block editor isn't initialized yet.
+	wp.data.subscribe( function() {
+
+		// Only check if we should register the block if it isn't already registered.
+		if ( isVolunteerManagementBlockRegistered === false ) {
+
+			// Grab the registered blocks.
+			let registeredBlocks                     = wp.data.select( 'core/blocks' ).getBlockTypes();
```

Comment:
Are these two lines of code needed? Don't we already have the `isVolunteerManagementBlockRegistered` variable to tell us that the block isn't registered yet?

Reply:
Ah great catch, not needed at all since I've declared the variable at the start of the block, and we're now conditionally registering the block.